lib/reek/code_comment.rb

```diff
@@ -20,19 +21,42 @@ class CodeComment
     DISABLE_DETECTOR_CONFIGURATION = '{ enabled: false }'.freeze
     MINIMUM_CONTENT_LENGTH         = 2
     LEGACY_SEPARATOR               = ':'.freeze
+    UNKNOWN_SMELL_DETECTOR_MESSAGE = <<-EOS.freeze
```

Comment:
I was just pointing out helpful syntax that I think many people aren't aware of. Basically you can write something like

```ruby
<<-ANY_HEREDOC
foo
bar
baz
ANY_HEREDOC
```

with single quotes wrapping the heredoc end of string token like so

```ruby
<<-'ANY_HEREDOC'
foo
bar
baz
ANY_HEREDOC
```

the benefit of this is that it signals that interpolation doesn't happen in the entire heredoc